rundeckapp/grails-app/assets/less/library/markdeep.less

```diff
@@ -156,9 +156,9 @@
 
 .md table.table {
   margin: auto;
-  .table-bordered();
-  .table-condensed();
-  .table-hover();
+ // .table-bordered();
```

Comment:
Have you seen what this looks like in a Markdeep/markdown UI somewhere in the app? 
table-bordered, -condensed, -hovered - all contribute to the way a table looks in markdown output. What we'd see here is a table w/o borders, cell padding, and no onHover effect (row highlighting).

Can you confirm that there's no UI regression please.

Reply:
There appears to be a regression since the 2.11.x days. The current behavior is the same as the 3.0.x version, so the regression was introduced in that transition.